package.json

```diff
@@ -13,14 +13,16 @@
   "author": "Ammar Vehbi",
   "license": "MIT",
   "dependencies": {
+    "@sendgrid/mail": "^6.4.0",
     "bcryptjs": "^2.4.3",
     "config": "^3.1.0",
     "express": "^4.17.1",
     "express-validator": "^5.3.1",
     "gravatar": "^1.8.0",
     "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.5.11",
-    "request": "^2.88.0"
+    "request": "^2.88.0",
+    "sendgrid": "^5.2.3"
```

Comment:
Since we have `@sendgrid/mail` above, I think we do not need this `sendgrid` library